compute/cloud-client/src/main/java/compute/images/CreateImageFromImage.java

```diff
@@ -78,12 +77,12 @@ public static Image createImageFromImage(String projectId, String sourceImageNam
       if (storageLocation != null) {
         imageResource.addStorageLocations(storageLocation);
       }
-      if (guestOsFeature != null) {
-        for (String feature : guestOsFeature) {
-          GuestOsFeature build = GuestOsFeature.newBuilder()
-                  .setType(feature)
-                  .build();
-          imageResource.addGuestOsFeatures(build);
+      if (guestOsFeatures != null) {
+        for (String feature : guestOsFeatures) {
+          GuestOsFeature.Builder guestOsFeature = GuestOsFeature.newBuilder()
```

Comment:
```suggestion
          GuestOsFeature.Builder guestOsFeatureBuilder = GuestOsFeature.newBuilder()
```

Reply:
It's confusing to have `guestOsFeatures` and `guestOsFeature` denote different data types. Update this across snippets.